CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Version 4.4.6 (Under development)
 
+### AppCenter
+
+* **[Fix]** Fixed accounting of large payload files when saving logs locally
```

Comment:
It isn't clear what fixed from customer's point of view. Please re-word it with mention of max storage size limit